DomainServices/Services/CustomerService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using DomainModels;
+using DomainServices.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DomainServices.Services
+{
+    public class CustomerService : ICustomerService
+    {
+        private readonly List<Customer> _customers = new();
+
+        public virtual void Create(Customer customer)
+        {
```

Comment:
[Comentário Acima](https://github.com/LuanFortunatoW/EverestBackendDotnet/pull/2/files#r1027900637)
```suggestion
        public void Create(Customer customer)
        {
```